### DISCUSSION TITLE
can I do this update without doing a raw query?

### DISCUSSION THREAD

**Discussion by massivefermion** [prisma/prisma#10369] [Q&A] [answered]
I'm using postgresql and I have a table with a boolean field. I need to toggle this boolean field based on user requests. This is what I have now:

```sql
prsima.$queryRaw`update "Profile" set "emailVisibility" = not "emailVisibility" where "userId" = ${userId} returning *`
```

But I'm wondering whether I really need to do a raw query. Does prisma provide the ability to update a column based on the current value of that column? Or more generally, any other column?

Thanks

**Comment by TasinIshmam** [comment, marked as answer]
Hey @massivefermion :wave:  

Unfortunately, it is not possible to toggle a field in Prisma at the moment. What you're doing now seems like the best workaround.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
